app/app/routing/graphhopper.py

```diff
@@ -0,0 +1,28 @@
+import requests
+from app.app.secret import GRAPHHOPPER_KEY
```

Comment:
can we inject this into the Graphhopper constructor, so it would be called in a different class and it doesn't crash because of the import in Travis anymore? Instead we can insert a fake KEY for the test... that should work if we push the YAML with this fake key I think, WDYT?